olp-cpp-sdk-core/include/olp/core/geo/coordinates/GeoCoordinates.h

```diff
@@ -56,8 +56,9 @@ class CORE_API GeoCoordinates {
    * [-90, 90] range.
    * @param longitude_degrees WGS84 longitude in degrees. Valid values are in
    * [-180, 180) range. Use normalized() to put geo coordinate in a valid range.
+   * @param degrees a struct to differ from the constructor with radians
```

Comment:
Not needed. The official notion is *tag*, resp *tag dispatching*
https://www.boost.org/community/generic_programming.html#tag_dispatching
But i would omit it, except doxygen throws an error then maybe use following

```suggestion
   * @param Dispatch tag for degree coordinates.
```